ios/RNVICallModule.m

```diff
@@ -158,6 +164,52 @@ @implementation RNVICallModule
     }
 }
 
+RCT_EXPORT_METHOD(startReceiving:(NSString *)streamId
+                        resolver:(RCTPromiseResolveBlock)resolve
+                        rejecter:(RCTPromiseRejectBlock)reject) {
+    VIRemoteVideoStream *remoteVideoStream = [RNVICallManager getRemoteVideoStreamById:streamId];
+    if (remoteVideoStream) {
+        [remoteVideoStream startReceivingWithCompletion:^(NSError * _Nullable error) {
+            if (error) {
+                reject(nil, error, nil);
```

Comment:
error code should not be null

the same applies to `stopReceiving`, `requestVideoSize` methods

Reply:
done